### PR TITLE
Add permission for new git-bisect plugin

### PR DESCRIPTION
**Pull request by dorav**
# Description

[This is the hosting request ](https://issues.jenkins-ci.org/browse/HOSTING-402)
[This is the git repository](https://github.com/jenkinsci/git-bisect-plugin)

# Submitter checklist for changing permissions

### Always

- [ ] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [ ] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [ ] Make sure the file is created in `permissions/` directory
- [ ] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [ ] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [ ] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [ ] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [ ] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [ ] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

**Comment by daniel-beck** [comment]
https://github.com/jenkinsci/git-bisect-plugin

File: permissions/plugin-git-bisect.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: "git-bisect"
+paths:
+- "org/jenkins-ci/plugins/git-bisect"
```
Comment (daniel-beck) [inline permissions/plugin-git-bisect.yml]: Does not match pom.xml. Please read this repository's readme and adapt either this file, or the pom.xml.

Comment (dorav) [reply permissions/plugin-git-bisect.yml]: Which field does not match? I'm probably looking at the wrong field..

Comment (orrc) [reply permissions/plugin-git-bisect.yml]: You've specified a `groupId` of `git`, which overrides the parent `groupId` of `org.jenkins-ci.plugins`.

The simplest fix would be to delete the `<groupId>git</groupId>` line in your POM.

Comment (dorav) [reply permissions/plugin-git-bisect.yml]: Thanks. I had a feeling that it shouldn't be there, had no idea what it does.

Iv'e removed it.

**Pull request merged**